client/components/RelatedProducts/Related/RelatedProductsList.jsx

```diff
@@ -10,16 +8,15 @@ import Typography from '@material-ui/core/Typography';
 //   visibility: hidden;
 // }
 
-const RelatedProductsList = ({relatedWithNameCatPrice}) => {
+const RelatedProductsList = ({relatedProductsData}) => {
 
   return  (
     <>
       <Typography>
         RELATED PRODUCTS
       </Typography>
       <Carousel showEmptySlots itemsToShow={4}>
-          {/* {products.products.map(item => <RelatedProductCard key={item.id} relatedWithNameCatPrice={item}/>)} */}
-          {relatedWithNameCatPrice.map(item => <RelatedProductCard key={item.id} relatedWithNameCatPrice={item}/>)}
+          {relatedProductsData.map(item => <RelatedProductCard key={item.id} relatedProductsData={item}/>)}
```

Comment:
You're using a prop here called ```relatedProductsData``` but it's the data for only one product

Reply:
Good call.  Will update.